app/app.js

```diff
@@ -63,6 +63,29 @@ function (app, $, _, Backbone, Bootstrap, Helpers, Utils, FauxtonAPI, Couchdb) {
   // Localize or create a new JavaScript Template object
   var JST = window.JST = window.JST || {};
 
+  var parseCookies = function (cookies) {
+    var list = {}, rc = cookies;
+    rc && rc.split(';').forEach(function (cookie) {
+      var parts = cookie.split('=');
+      list[parts.shift().trim()] = decodeURI(parts.join('='));
+    });
+    return list;
+  };
+
+  $._ajax = $.ajax;
+
+  $.ajax = function (settings) {
```

Comment:
Instead of overriding `$.ajax` could we rather use a [global setting](https://api.jquery.com/jQuery.ajaxSetup/)?